test/e2e_kubeadm/bootstrap_token_test.go

```diff
@@ -54,8 +57,12 @@ var _ = Describe("bootstrap token", func() {
 
 		tokenNum := 0
 		for _, s := range secrets.Items {
-			if s.Type == corev1.SecretTypeBootstrapToken {
-				//TODO: might be we want to further check tokens  (auth-extra-groups, usage etc)
+			// check extra group and usage of token, make sure at least one token exist
+			if s.Type == corev1.SecretTypeBootstrapToken && string(s.Data[bootstrapTokensAuthExtraGroup]) == bootstrapTokensGroup {
+				usageBootstrapAuthentication := string(s.Data[bootstrapTokensUsageAuthentication])
```

Comment:
instead of:
```
someVar := string(s.Data[key])
gomega.Expect(someVar).Should(gomega.Equal("true"), "..."
```

why not use?
```
_, existsKeyName := s.Data[keyName]
gomega.Expect(existsKeyName).Should(gomega.Equal(true), "..."
```